packages/udes-ui-validation/src/validation.tsx

```diff
@@ -0,0 +1,170 @@
+import validator from "validator";
+
+/*
+   Based on code found at
+https://medium.com/code-monkey/client-side-form-validation-in-react-40e367de47ba
```

Comment:
Indentation

Reply:
if you indent this more than it is it runs over the max line length, this seemed preferable, but I can tell it to ignore the restriction or put the URL on two lines (which to me is absolutely terrible ux but happy to do it)